backend/telemetry_shard/src/main.rs

```diff
@@ -91,6 +96,12 @@ struct Opts {
     /// on the machine. If no value is given, use an internal default that we have deemed sane.
     #[structopt(long)]
     worker_threads: Option<usize>,
+    /// Roughly how long to wait in seconds for new telemetry data to arrive from a node. If
+    /// telemetry for a node does not arrive in this time frame, we remove the corresponding node
+    /// state, and if no messages are received on the connection at all in this time, it will be
+    /// dropped.
+    #[structopt(long, default_value = "30")]
```

Comment:
I think we might need a longer time here to account for nodes that are "legitimately silent" due to averse network conditions. Tbh I'm actually not sure if stuck nodes send telemetry messages or not, i.e. if it's possible/plausible that a node stays silent because it's having internal issues of some sort?